pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -714,6 +717,10 @@ func (c *Controller) createIPSecTunnelPort(nodeName string, nodeIP net.IP) (int3
 		// Let NodeRouteController retry at errors.
 		return 0, fmt.Errorf("failed to get of_port of IPsec tunnel port for Node %s", nodeName)
 	}
+	// Set external_ids to the port for upgrade case.
+	if err := c.ovsBridgeClient.SetPortExternalIDs(portName, ovsExternalIDs); err != nil {
```

Comment:
initInterfaceStore has a logic to fill missed AntreaInterfaceTypeKey, I think this could be moved there to unify the logic.

Reply:
Done. Thanks!